src/tokens.json

```diff
@@ -1,1288 +1,18 @@
 [
```

Comment:
You should put a readme file explaining what the criteria is to make it on the list.